### PR TITLE
Use multi-arch pause image for tests

### LINKED ISSUE #57588: kubernetes conformance tests are failing for other than x86 archs
**Is this a BUG REPORT or FEATURE REQUEST?**:

> Uncomment only one, leave it on its own line: 
>
/kind bug
> /kind feature

**What happened**:
Failures - https://k8s-testgrid.appspot.com/sig-node-ppc64le#conformance
Part of https://github.com/kubernetes/kubernetes/pull/56716 pause is not used properly for tests for other arches than x86. As the current pause image is not a manifest image, so we need to use a different image for different archs like `pause-amd64` for `amd64`, `pause-ppc64le` for`ppc64le` etc..

With this, I'm using logs-generator from `gcr.io/kubernetes-e2e-test-images` repo
**What you expected to happen**:

**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**:

**Environment**:
- Kubernetes version (use `kubectl version`):
- Cloud provider or hardware configuration:
- OS (e.g. from /etc/os-release):
- Kernel (e.g. `uname -a`):
- Install tools:
- Others:

### PR DESCRIPTION
**Pull request by mkumatag**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #57588

**Special notes for your reviewer**:

**Release note**:

```NONE```

### CODE REVIEW

**Review by luxas — APPROVED** [review]
/lgtm

I think this probably should be a cherrypick candidate for v1.9 to fix e2e tests on some platforms there

**Comment by luxas** [comment]
@mkumatag Could you please create an OWNERS file for `test/utils/image` that have myself and @ixdy as approvers and reviewers as well as you as a reviewer?

**Comment by mkumatag** [comment]
@luxas created a https://github.com/kubernetes/kubernetes/pull/57634 for the same.
> @mkumatag Could you please create an OWNERS file for test/utils/image that have myself and @ixdy as approvers and reviewers as well as you as a reviewer?
> 
> (test/ approval is not needed for this change)

File: test/utils/image/manifest.go [hunk 1]
```diff
@@ -83,7 +83,7 @@ var (
 	NoSnatTestProxy          = ImageConfig{e2eRegistry, "no-snat-test-proxy", "1.0", true}
 	NWayHTTP                 = ImageConfig{e2eRegistry, "n-way-http", "1.0", true}
 	// When these values are updated, also update cmd/kubelet/app/options/options.go
-	Pause               = ImageConfig{gcRegistry, "pause", "3.0", false}
+	Pause               = ImageConfig{gcRegistry, "pause", "3.0", true}
```
Comment (ixdy) [inline test/utils/image/manifest.go]: this should probably be `pause:3.1` now?

oh, or this is just being updated in #57517.  cc @verb

**Pull request merged**